explore-analyze/discover/discover-get-started.md

```diff
@@ -222,6 +222,10 @@ One of the unique capabilities of **Discover** is the ability to combine free te
 
 ![Search field in Discover](/explore-analyze/images/kibana-discover-search-field.png "")
 
+:::{note}
+Free text searches that don't specify a field may not return expected results depending on how the [`index.query.default_field` index setting](elasticsearch://reference/elasticsearch/index-settings/index-modules.md#index-query-default-field) is configured for the indices matching the current data view.
+:::
```

Comment:
@ninoslavmiskovic does this make sense regarding https://github.com/elastic/enhancements/issues/18024?